person.rb

```diff
@@ -1,14 +1,21 @@
+require './nameable'
+
 class Person
   def initialize(age, name = 'Unknown', parent_permission: true)
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @corrector = Nameable.new
   end
 
   attr_accessor :name, :age
   attr_reader :id
 
+  def correct_name
+    @name = @corrector.correct_name(name)
+  end
```

Comment:
- [ ] The `correct_name` method in this **Person** class should return only `name`